pkg/pkg.bzl

```diff
@@ -503,68 +504,171 @@ def pkg_tar(name, **kwargs):
         ),
-        "architecture": attr.string(
-            default = "all",
-            doc = """Package architecture. Must not be used with architecture_file.""",
+        "maintainer": attr.string(
+            doc = "The maintainer of the package.",
+            mandatory = True,
+        ),
+        "version": attr.string(
+            doc = """Package version. Must not be used with `version_file`.""",
         ),
-        "distribution": attr.string(default = "unstable"),
-        "urgency": attr.string(default = "medium"),
-        "maintainer": attr.string(mandatory = True),
-        "preinst": attr.label(allow_single_file = True),
-        "postinst": attr.label(allow_single_file = True),
-        "prerm": attr.label(allow_single_file = True),
-        "postrm": attr.label(allow_single_file = True),
-        "config": attr.label(allow_single_file = True),
-        "templates": attr.label(allow_single_file = True),
-        "triggers": attr.label(allow_single_file = True),
-        "conffiles_file": attr.label(allow_single_file = True),
-        "conffiles": attr.string_list(default = []),
         "version_file": attr.label(
             doc = """File that contains the package version.
-            Must not be used with version.""",
+            Must not be used with `version`.""",
             allow_single_file = True,
         ),
-        "version": attr.string(
-            doc = """Package version. Must not be used with version_file.""",
-        ),
-        "description_file": attr.label(allow_single_file = True),
-        "description": attr.string(),
-        "built_using_file": attr.label(allow_single_file = True),
-        "built_using": attr.string(),
-        "priority": attr.string(),
-        "section": attr.string(),
-        "homepage": attr.string(),
-        "depends": attr.string_list(default = []),
-        "depends_file": attr.label(allow_single_file = True),
-        "suggests": attr.string_list(default = []),
-        "enhances": attr.string_list(default = []),
-        "breaks": attr.string_list(default = []),
-        "conflicts": attr.string_list(default = []),
-        "predepends": attr.string_list(default = []),
-        "recommends": attr.string_list(default = []),
-        "replaces": attr.string_list(default = []),
-        "provides": attr.string_list(default = []),
+        "config": attr.label(
+            doc = """config file used for debconf integration.
+            See https://www.debian.org/doc/debian-policy/ch-binary.html#prompting-in-maintainer-scripts.""",
+            allow_single_file = True,
+        ),
+        "description": attr.string(
+            doc = """The package description. Must not be used with `description_file`.""",
+        ),
+        "description_file": attr.label(
+            doc = """The package description. Must not be used with `description`.""",
+            allow_single_file = True
+        ),
+        "distribution": attr.string(
+            doc = "TBD",
```

Comment:
These TBD docstrings should probably be set somewhere, methinks

Reply:
I pointed to the debian docs.